docs/pages/kubernetes-access/helm/guides/digitalocean.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: Get started with Teleport on DigitalOcean Managed Kubernetes cluster.
+description: Learn how to get started with Teleport on DigitalOcean Managed Kubernetes cluster
+---
+
+This guide will show you how to get started with Teleport on DigitalOcean managed Kubernetes cluster.  Specifically, we will 
```

Comment:
Updated "DigitalOcean managed Kubernetes cluster" to "DigitalOcean Kubernetes". Regarding the list, I think it should be numbered list because it presents the primary steps for the tutorial.